atcoder/abc157B_bingo.d

```diff
@@ -0,0 +1,48 @@
+import std.stdio: writeln, writefln, readln;
```

Comment:
add a comment with author and problem info

Reply:
Ok, I'll do it